salt/journal/config/etc-nginx-sites-available-journal.conf

```diff
@@ -375,6 +375,54 @@ server {
     }
     {% endif -%}
 
+    # ali-tm-amin@2024-10-30 proxy journal hosted pages to hubspot
+    {% if pillar.journal.community_newsletter_url %}
+    # Proxy /community-newsletter pages to hubspot
+    location /content-alerts/early-career {
+        proxy_pass {{ pillar.journal.community_newsletter_url }};
+        proxy_ssl_server_name on;
+        proxy_connect_timeout 10s;
+        proxy_read_timeout 10s;
+        proxy_set_header X-HubSpot-Client-IP $proxy_protocol_addr;  # For PROXY protocol
+    }
+    {% endif -%}
+
+    {% if pillar.journal.elife_newsletter_url %}
+    # Proxy /elife-newsletter pages to hubspot
+    location /content-alerts/elife-newsletter {
+        proxy_pass {{ pillar.journal.elife_newsletter_url }};
+        proxy_ssl_server_name on;
+        proxy_connect_timeout 10s;
+        proxy_read_timeout 10s;
+        proxy_set_header X-HubSpot-Client-IP $proxy_protocol_addr;  # For PROXY protocol
+    }
+    {% endif -%}
+
+    {% if pillar.journal.content_alerts_url %}
+    # Proxy /content-alerts pages to hubspot
+    location /content-alerts {
+        proxy_pass {{ pillar.journal.content_alerts_url }};
+        proxy_ssl_server_name on;
+        proxy_connect_timeout 10s;
+        proxy_read_timeout 10s;
+        proxy_set_header X-HubSpot-Client-IP $proxy_protocol_addr;  # For PROXY protocol
+    }
+    {% endif -%}
+
+    {% if pillar.journal.community_newsletter_url or pillar.journal.elife_newsletter_url or pillar.journal.content_alerts_url %}
+
+    location ~ ^/dist/(.*)$ {
```

Comment:
I'm not seeing any `/dist` resources, and it would conflict with pubpub if it did. However, I am seeing a complex patchwork of other resource requests coming in, so we should take a look at how many are essential and that we need to manage.